packages/cojson/src/coValues/coList.ts

```diff
@@ -338,22 +374,19 @@ export class RawCoList<
 
       // We navigate the predecessors before processing the current opID in the list
       if (shouldTraversePredecessors) {
-        for (let i = entry.predecessors.length - 1; i >= 0; i--) {
-          todo.push(entry.predecessors[i]!);
+        for (const predecessor of entry.predecessors) {
+          todo.push(predecessor);
```

Comment:
Changed the iteration order, to align with the splice -> push change done above.